ts/torch_handler/base_handler.py

```diff
@@ -278,6 +303,9 @@ def inference(self, data, *args, **kwargs):
         with torch.no_grad():
             marshalled_data = data.to(self.device)
             results = self.model(marshalled_data, *args, **kwargs)
+            if torch_xla_enabled:
+                xm.mark_step()
```

Comment:
not super familiar with xla internals but what does this line do?

Reply:
Removed the `xm.mark_step()` cause this is essential for training, optional for inferencing. In short, the value /calculation is upon either a `xm.mark_step()` or when it gets retrieved. In our case it's the latter one.